src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,40 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+
+  constructor(hidden, health) {
+    super(hidden, health);
+    this.hidden = false;
+  }
```

Comment:
Your super constructor accepts only name, so you need to only pass it as parameter.
```suggestion
  constructor(name) {
    super(name);
    this.hidden = false;
  }
```